feat: Add Field component (#1228)
Amsterdam/design-system: packages/css/src/components/field/README.md

<!-- @license CC0-1.0 -->

# Field

> Wraps around a single input and can indicate if that input has a validation error.

Reviewer: ```suggestion
A field wraps around an input and may indicate if that input has a validation error.
```

I'm not sure about adding “A field wraps [...]” before the sentence. I see that we are inconsistent with repeating the component's (generic) name in the summary to have full sentences. Often we do start the description with the component's generic or capitalised name, but not always: i.e. “An Alert informs the user [...]” or “The accordion component is a [...]”.
